helmfile.d/stacks/monitoring-prometheus.yaml

```diff
@@ -14,9 +14,11 @@ templates:
     needs:
       - monitoring/networkpolicy
       - monitoring/podsecuritypolicy
+      {{- if .Values | get "ck8sManagementCluster.enabled" false }}
```

Comment:
If, like @Zash is questioning, this is needed. Can we combine the two if statements with an `and` instead of having 2?

Reply:
Removed the check as it was redundant

Reply:
Added the check back and combined it into one: https://github.com/elastisys/compliantkubernetes-apps/pull/2372/commits/17466d0e9d95e733e1eb6ca066e61259844f737b